src/config.ts

```diff
@@ -108,3 +108,7 @@ export function getZendeskGroupIdPublic(): number {
 export function getAnalyticsCookieDomain(): string {
   return process.env.ANALYTICS_COOKIE_DOMAIN;
 }
+
+export function getServiceDomain(): string {
+  return process.env.SERVICE_DOMAIN;
```

Comment:
Is there any sensible default value here? Or does `undefined` work okay?

Reply:
good question, let me add an additional check in the next PR